plugins/copr.py

```diff
@@ -529,6 +561,22 @@ def _disable_repo(self, copr_username, copr_projectname):
             self.base.conf.write_raw_configfile(repo.repofile, multilib_repo_id,
                                                 self.base.conf.substitutions, {"enabled": "0"})
 
+        # disable runtime dependencies as well
+        repo_file = ConfigParser(strict=False)
+        repo_file.read(repo.repofile)
+        repo_changed = False
+
+        for section in repo_file.sections():
+            if "Runtime dependency" not in repo_file.get(section, "name"):
+                continue
+            repo_file.set(section, "enabled", "0")
+            repo_changed = True
+
+        if repo_changed:
+            with open(repo_filename, "w") as repo:
+                repo_file.write(repo)
```

Comment:
This is still really, really ugly.  What if we simply added - on copr fe side - something
like `copr_dep_of=copr:safdsfdasd:asdfasdf`, and - instead of another ConfigParser
hack went through all the `if multilib_repo_id in self.base.repos` which have
`copr_dep_of=<COPR_ID>`?

Reply:
I mean, we could disable the repo the same way as we do with the normal repo.

Reply:
I was told by @j-mracek that it will throw some warnings into `dnf -v` output,
but I don't view this as huge problem.  Same as we can have several warnings
because of duplicate repo ID enabled as a result of dep hell.

We could have another config file with dependency tree (I don't know whether
@j-mracek thought this in 1:1 irc chat).  The question is whether it's worth it
in this case (my guess is no).

Reply:
Simply speaking, id much rather iterate over all repositories - and disable those
which are special - then reading the config file again by ConfigParser, and
parse it again.

Reply:
Using a repo file for plugin configuration or external data is not good idea for long or final solution, but it is fine as a prove of concept.

.repo files have some disadvantages. They are difficult to update, because they are editable by an end user. After modification they cannot be easily updated by rpm. In case of origin from plugin (copr, substriction-manager) they are updated but with lose of information provided by user or not updated at all.

.repo files have also some advantages. They are loaded before metadata therefore the set of required repositories can be adjusted.

Also implementation of the functionality only to DNF is not suitable as a long term solution. Think like that you want to create a new industry standard and not only to enhance DNF plugin.